### PR TITLE
Adding support to Debian Jessie and using exiv2-role v1.0.1

### PR DESCRIPTION
**Pull request by dortegau**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change

- Adding support to Debian Jessie
- Using [exiv2-role v1.0.1](https://github.com/idealista/exiv2-role/releases/tag/1.0.1) to fix broken Exiv2 download link issue.

### Applicable Issues

#14 
#15

### CODE REVIEW

**Comment by jnogol** [comment]
`CHANGELOG.md` should be updated, I guess

File: molecule.yml [hunk 1]
```diff
@@ -31,7 +32,7 @@ docker:
     - name: yatm.cont
       ansible_groups:
         - yatm
-      image: geerlingguy/docker-debian9-ansible
+      image: geerlingguy/docker-debian8-ansible
```
Comment (jnogol) [inline molecule.yml]: Maybe both containers, to ensure Debian 8 & 9 compatibility?

Comment (dortegau) [reply molecule.yml]: AFAIK you cannot use a specific container in molecule 1.x.x if you have more than one. 

Could you give me an example?

Comment (jnogol) [reply molecule.yml]: My idea is starting up both containers and test the role against both of them:

``` yaml
docker:
  containers:
    - name: yatm_debian9.cont
      ansible_groups:
        - yatm
      image: geerlingguy/docker-debian9-ansible
      image_version: latest

      privileged: True
      cap_add:
        - SYS_ADMIN
      volume_mounts:
        - '/sys/fs/cgroup:/sys/fs/cgroup:ro'
      command: '/lib/systemd/systemd'

    - name: yatm_debian8.cont
      ansible_groups:
        - yatm
      image: geerlingguy/docker-debian8-ansible
      image_version: latest

      privileged: True
      cap_add:
        - SYS_ADMIN
      volume_mounts:
        - '/sys/fs/cgroup:/sys/fs/cgroup:ro'
      command: '/lib/systemd/systemd'
```

Comment (dortegau) [reply molecule.yml]: good idea! done 👍

**Pull request merged**